analyzere/base_resources.py

```diff
@@ -148,6 +149,13 @@ def to_dict(self):
             d[k] = to_dict(v)
         return d
 
+    def __hash__(self):
+        # Certainly not the most efficient hash, since anything without an ID
+        # has the same hash value.
+        if hasattr(self, "id"):
+            int(uuid.UUID(self.id))
+        return 0
```

Comment:
@DanielArndt according to the docs (https://docs.python.org/3.4/glossary.html#term-hashable) Python uses `id()` for it's default `__hash__` implementation, would that be suitable here?